apigw-lambda-dynamodb-sam-go/example-pattern.json

```diff
@@ -0,0 +1,44 @@
+{
+    "title": "API Gateway to AWS Lambda to Amazon DynamoDB with Go and AWS SAM",
+    "description": "This pattern uses AWS SAM to deploy an API Gateway REST API connected to an AWS Lambda function, wirtten in Go that can put new items in an Amazon DynamoDB table.",
+    "language": "Go",
+    "level": "200",
+    "framework": "SAM",
+    "introBox": {
+      "headline": "How it works",
+      "text": [""]
```

Comment:
You can optionally add some info here from README

Reply:
added